src/views/figure_editable.ts

```diff
@@ -180,7 +182,7 @@ export class FigureEditableView extends FigureView<
       const componentProps: FigureOptionsProps = {
         can,
         files,
-        modelMap: this.props.getModelMap(),
+        filesMap: encode(this.view.state.doc),
```

Comment:
Do we need to encode the entire doc here? Can we filter out the nodes we don't need?

Reply:
updated to create a lighter manuscript node with a graphical abstract section and a section containing all figureElement in other sections